lib/models/cac_location.rb

```diff
@@ -48,15 +48,30 @@ class CACLocation
   }.freeze
 
   attr_accessor(*ATTRIBUTES)
+  attr_writer :componentized_us_address
 
   # Returns an array of CACLocations from the API.
   #
   # @return [Array] all CACLocations from the API
+  # rubocop:disable Metrics/MethodLength
   def self.all_from_api
-    TestSites::CAC.cac_data.map do |hash|
-      CACLocation.new(hash)
+    geocoder_results = TestSites::GeocoderResults.new.filtered
+    locations = []
+
+    TestSites::CAC.cac_data.each do |json|
```

Comment:
small nitpick on naming - it's a hash (actually a mash), not json

Reply:
also, why not use `map` rather than incrementally appending to an array within an `each`?